impeller/aiks/aiks_unittests.cc

```diff
@@ -1614,6 +1614,172 @@ TEST_P(AiksTest, SolidColorCirclesOvalsRRectsMaskBlurCorrectly) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+struct MaskBlurTestConfig {
+  FilterContents::BlurStyle style = FilterContents::BlurStyle::kNormal;
+  Scalar sigma = 1.0f;
+  Scalar alpha = 1.0f;
+  std::shared_ptr<ImageFilter> image_filter;
+  bool invert_colors = false;
+  BlendMode blend_mode = BlendMode::kSourceOver;
+};
+
+static Picture MaskBlurVariantTest(const AiksTest& test_context,
```

Comment:
Oh wait, can you put these in aiks_blur_unittests, please?  I wish I had a better way to force the sorting of thins.

Reply:
Done.